daml-script/daml3/Daml/Script/Internal.daml

```diff
@@ -31,6 +31,7 @@ module Daml.Script.Internal
   , -- Exceptions
     -- TODO: consider exposing this normally.
     tryToEither
+  , throwAnyException
```

Comment:
Decided that `throwAnyException` should live in `Daml.Script.Internal`, its reasonable to expose for our own tools

Reply:
doesn't `throw` do the same thing?
`throw : Exception e => e -> m t` 
(`throw @Script @AnyException : AnyException -> Script t`)

Reply:
Ah! I hadn't realised we have `HasToAnyException AnyException`

Reply:
Oh i remember why, `AnyException` doesn't have `HasThrow`, and Exception is defined as
```